docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
     volumes:
     - ${DESECSTACK_WWW_CERTS}:/etc/ssl/private:ro
     - ./www/html:/usr/share/nginx/html:ro
+    environment:
+    - DESECSTACK_DOMAIN
```

Comment:
They do overlap, in fact we have

    DESECSTACK_API_ALLOWED_HOSTS = desec.DESECSTACK_DOMAIN update.dedyn.DESECSTACK_DOMAIN update6.dedyn.DESECSTACK_DOMAIN

Adopted `README.md`, `.env.default`, `settings.py`, `docker-compose.yml` to reflect that fact. `DESECSTACK_API_ALLOWED_HOSTS` is not needed anymore.

A disadvantage of this clean cut is a dependency on desec-io/desec-www@9485f5b, any previous version won't work due to missing `DESECSTACK_API_ALLOWED_HOSTS` provided to the `api` environment.